Index: b/api-server/pscheduler-api-server.conf.raw
===================================================================
--- a/api-server/pscheduler-api-server.conf.raw	2023-11-10 15:29:47.805480253 -0500
+++ b/api-server/pscheduler-api-server.conf.raw	2023-11-10 15:29:49.598886923 -0500
@@ -27,7 +27,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Require all granted
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
