Index: b/api-server/pscheduler-api-server.conf.raw
===================================================================
--- a/api-server/pscheduler-api-server.conf.raw	2024-06-06 11:39:09.686923766 -0400
+++ b/api-server/pscheduler-api-server.conf.raw	2024-06-06 11:39:12.882853941 -0400
@@ -27,7 +27,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Require all granted
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
