Index: b/api-server/pscheduler-api-server.conf.raw
===================================================================
--- a/api-server/pscheduler-api-server.conf.raw	2025-03-19 12:32:16.622608277 -0400
+++ b/api-server/pscheduler-api-server.conf.raw	2025-03-19 12:32:20.113401305 -0400
@@ -28,7 +28,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Require all granted
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
