Index: b/api-server/pscheduler-api-server.conf.raw
===================================================================
--- a/api-server/pscheduler-api-server.conf.raw	2024-04-04 08:06:32.278077847 -0400
+++ b/api-server/pscheduler-api-server.conf.raw	2024-04-04 08:06:35.717270074 -0400
@@ -27,7 +27,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Require all granted
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
