Index: b/utilities/backup.raw
===================================================================
--- a/utilities/backup.raw	2023-11-10 15:29:47.813622299 -0500
+++ b/utilities/backup.raw	2023-11-10 15:29:49.815001665 -0500
@@ -73,6 +73,7 @@
     --format plain \
     --serializable-deferrable \
     --user pscheduler \
+    --host localhost \
     pscheduler \
     > "${WORK_DIR}/database-dump"
 $INTERACTIVE && printf ' Done.\n' 1>&2 || true
Index: b/utilities/restore.raw
===================================================================
--- a/utilities/restore.raw	2023-11-10 15:29:47.813819133 -0500
+++ b/utilities/restore.raw	2023-11-10 15:29:49.815298167 -0500
@@ -203,7 +203,7 @@
 DB_OUTPUT="${WORK_DIR}/db-output"
 LOAD_EXIT=0
 
-postgresql-load "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
+su -l -c psql postgres < "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
     || LOAD_EXIT=$?
 
 if [ "${LOAD_EXIT}" -ne 0 ]
