diff -Nru database/db-change-password.raw database-debian/db-change-password.raw
--- pscheduler-server/database/db-change-password.raw	2022-03-15 17:30:00.000000000 -0500
+++ pscheduler-server/database/db-change-password.raw	2022-03-15 17:30:01.000000000 -0500
@@ -22,19 +22,26 @@
 [ -t 0 -a -t 1 -a -t 2 ] && INTERACTIVE=true || INTERACTIVE=false


-DATABASE="__DATABASE__"
-ROLE="__ROLE__"
-PASSWORDFILE="__PASSWORDFILE__"
-DSNFILE="__DSNFILE__"
-PGPASSFILE="__PGPASSFILE__"
+DATABASE=pscheduler
+ROLE=pscheduler
+DATABASE_DIR=/etc/pscheduler/database
+PASSWORDFILE="${DATABASE_DIR}/database-password"
+DSNFILE="${DATABASE_DIR}/database-dsn"
+PGPASSFILE="${DATABASE_DIR}/pgpassfile"


 # Generate the new password and write it to the config files
 @@ -84,7 +85,13 @@
  tr -d "\n" < "${PASSWORDFILE}" >> "${ROLESQL}"
   printf "';\n"  >> "${ROLESQL}"

-postgresql-load "${ROLESQL}"
+DB_OUTPUT="${TMPBASE}.output"
+< "${ROLESQL}" su -l -c 'psql -U "${ROLE}" "pscheduler"' postgres > "${DB_OUTPUT}" 2>&1
+if [ $? -ne 0 ]
+then
+    cat "${OUTPUT}" 1>&2
+    exit 1
+fi

 if $INTERACTIVE
  then


