Index: b/database/warmboot.raw
===================================================================
--- a/database/warmboot.raw	2023-11-10 15:29:47.811490995 -0500
+++ b/database/warmboot.raw	2023-11-10 15:29:49.671437201 -0500
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
