Index: b/database/warmboot.raw
===================================================================
--- a/database/warmboot.raw	2025-03-19 12:32:16.632404754 -0400
+++ b/database/warmboot.raw	2025-03-19 12:32:21.185672624 -0400
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
