Index: b/database/warmboot.raw
===================================================================
--- a/database/warmboot.raw	2023-03-28 19:35:51.323238000 +0200
+++ b/database/warmboot.raw	2023-04-12 16:17:23.794928318 +0200
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
