Index: b/database/warmboot.raw
===================================================================
--- a/database/warmboot.raw	2023-04-13 16:49:35.561045608 -0400
+++ b/database/warmboot.raw	2023-04-13 16:49:37.642675718 -0400
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
