Index: b/database/warmboot.raw
===================================================================
--- a/database/warmboot.raw	2024-04-04 08:06:32.285531899 -0400
+++ b/database/warmboot.raw	2024-04-04 08:06:36.785985719 -0400
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
