Index: b/database/warmboot.raw
===================================================================
--- a/database/warmboot.raw	2024-06-06 11:39:09.695346537 -0400
+++ b/database/warmboot.raw	2024-06-06 11:39:13.620558525 -0400
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
