--- a/diags.raw	2020-02-24 14:22:21.555030000 +0000
+++ b/diags.raw	2020-02-24 14:32:25.003027000 +0000
@@ -98,7 +98,7 @@
 
 section "New Tasks by Hour Added"
 
-postgresql-load <<EOF
+su -l -c psql postgres <<EOF
 \c pscheduler
 SELECT
     date_trunc('hour', added) AS hour,
@@ -113,7 +113,7 @@
 
 section "New Runs by Hour Added"
 
-postgresql-load <<EOF
+su -l -c psql postgres <<EOF
 \c pscheduler
 SELECT
     date_trunc('hour', added) AS hour,
@@ -128,7 +128,7 @@
 
 section "Run States by Hour"
 
-postgresql-load <<EOF
+su -l -c psql postgres <<EOF
 \c pscheduler
 SELECT
     run_state.enum AS state,
@@ -187,7 +187,7 @@
 fi
 
 section "HTTPD Error Log (Last ${LOG_LIMIT} Lines)"
-LOG="/var/log/httpd/error_log"
+LOG="/var/log/apache2/error.log"
 if [ -r "${LOG}" ]
 then
     tail "-${LOG_LIMIT}" "${LOG}"
@@ -198,17 +198,11 @@
 
 section "PostgreSQL Log (Last ${LOG_LIMIT} Lines)"
 
-PG_DATA=$(postgresql-load <<EOF
-\\t
-\\a
-SELECT setting FROM pg_settings WHERE name = 'data_directory';
-EOF
-)
-PG_LOG="${PG_DATA}/pg_log"
+PG_LOG="/var/log/postgresql"
 
 if [ -d "${PG_LOG}" ]
 then
-    ls -rtd ${PG_DATA}/pg_log/* | xargs cat | tail "-${LOG_LIMIT}"
+    ls -rtd ${PG_LOG}/*.log | xargs cat | tail "-${LOG_LIMIT}"
 else
     echo "PostgreSQL logs not found in ${PG_LOG}"
 fi
