--- src/pl/plperl/GNUmakefile.old	2018-06-27 00:27:01.614958217 +0100
+++ src/pl/plperl/GNUmakefile	2018-06-27 00:27:15.660034493 +0100
@@ -54,6 +54,9 @@
 
 SHLIB_LINK = $(perl_embed_ldflags)
 
+# Force rpath to be used even though we disable it everywhere else
+SHLIB_LINK += $(rpath)
+
 REGRESS_OPTS = --dbname=$(PL_TESTDB) --load-extension=plperl  --load-extension=plperlu
 REGRESS = plperl plperl_lc plperl_trigger plperl_shared plperl_elog plperl_util plperl_init plperlu plperl_array plperl_call plperl_transaction
 # if Perl can support two interpreters in one backend,
