--- src/Makefile.global.in.old	2018-03-11 21:25:30.148541091 +0000
+++ src/Makefile.global.in	2018-03-11 21:26:00.365690496 +0000
@@ -83,7 +83,7 @@
 # Installation directories
 #
 # These are set by the equivalent --xxxdir configure options.  We
-# append "postgresql" to some of them, if the string does not already
+# append "pgsql" to some of them, if the string does not already
 # contain "pgsql" or "postgres", in order to avoid directory clutter.
 #
 # In a PGXS build, we cannot use the values inserted into Makefile.global
@@ -104,14 +104,14 @@
 datadir := @datadir@
 ifeq "$(findstring pgsql, $(datadir))" ""
 ifeq "$(findstring postgres, $(datadir))" ""
-override datadir := $(datadir)/postgresql
+override datadir := $(datadir)/pgsql
 endif
 endif
 
 sysconfdir := @sysconfdir@
 ifeq "$(findstring pgsql, $(sysconfdir))" ""
 ifeq "$(findstring postgres, $(sysconfdir))" ""
-override sysconfdir := $(sysconfdir)/postgresql
+override sysconfdir := $(sysconfdir)/pgsql
 endif
 endif
 
@@ -120,7 +120,7 @@
 pkglibdir = $(libdir)
 ifeq "$(findstring pgsql, $(pkglibdir))" ""
 ifeq "$(findstring postgres, $(pkglibdir))" ""
-override pkglibdir := $(pkglibdir)/postgresql
+override pkglibdir := $(pkglibdir)/pgsql
 endif
 endif
 
@@ -129,7 +129,7 @@
 pkgincludedir = $(includedir)
 ifeq "$(findstring pgsql, $(pkgincludedir))" ""
 ifeq "$(findstring postgres, $(pkgincludedir))" ""
-override pkgincludedir := $(pkgincludedir)/postgresql
+override pkgincludedir := $(pkgincludedir)/pgsql
 endif
 endif
 
@@ -138,7 +138,7 @@
 docdir := @docdir@
 ifeq "$(findstring pgsql, $(docdir))" ""
 ifeq "$(findstring postgres, $(docdir))" ""
-override docdir := $(docdir)/postgresql
+override docdir := $(docdir)/pgsql
 endif
 endif
 
