--- pscheduler.orig/tests/psdns_test.py	2019-06-25 12:48:07.159134277 +0000
+++ pscheduler/tests/psdns_test.py	2019-06-26 11:07:41.737507239 +0000
@@ -46,7 +46,7 @@
         if have_network:
             # these should be stable
             assert(ret.get('does-not-exist.internet2.edu') is None)
-            self.assertEqual(ret.get('dns.google'), '8.8.4.4')
+            self.assertIn(ret.get('dns.google'), ['8.8.4.4','8.8.8.8'])
 
         # ipv6
 
