Index: b/diags.raw
===================================================================
--- a/diags.raw	2025-07-01 14:53:12.905693322 -0400
+++ b/diags.raw	2025-07-01 14:53:20.030172006 -0400
@@ -231,7 +231,7 @@
 
 
 section "HTTPD Error Log (Last ${LOG_LIMIT} Lines)"
-LOG="/var/log/httpd/error_log"
+LOG="/var/log/apache2/error.log"
 if [ -r "${LOG}" ]
 then
     tail "-${LOG_LIMIT}" "${LOG}"
@@ -254,17 +254,11 @@
 
 section "PostgreSQL Log (Last ${LOG_LIMIT} Lines)"
 
-PG_DATA=$(postgresql-load <<EOF
-\\t
-\\a
-SELECT setting FROM pg_settings WHERE name = 'data_directory';
-EOF
-)
-PG_LOG="${PG_DATA}/pg_log"
+PG_LOG="/var/log/postgresql"
 
 if [ -d "${PG_LOG}" ]
 then
-    ls -rtd ${PG_DATA}/pg_log/* | xargs cat | tail "-${LOG_LIMIT}"
+    ls -rtd ${PG_LOG}/*.log | xargs cat | tail "-${LOG_LIMIT}"
 else
     echo "PostgreSQL logs not found in ${PG_LOG}"
 fi
