Index: b/setup.py
===================================================================
--- a/setup.py	2025-10-29 14:58:40.903280019 +0100
+++ b/setup.py	2026-02-15 22:05:41.987421794 +0100
@@ -17,17 +17,6 @@
         'pscheduler.limitprocessor.identifier',
         'pscheduler.limitprocessor.limit',
     ],
-    install_requires=[
-        'dnspython >= 1.12.0',
-        'requests >= 2.6.0',
-        'psycopg2 >= 2.6.2',
-        'jsonschema >= 2.5.1',
-        'pyjq >= 2.2.0',
-        'python-dateutil >= 2.5.3',
-        'netifaces >= 0.5',
-        'ipaddr >= 2.1.9',
-    ],
     include_package_data=True,
     package_data={'pscheduler.limitprocessor': ['*.json']},
 
