Index: b/setup.py
===================================================================
--- a/setup.py	2026-02-16 20:52:16.405210830 +0100
+++ b/setup.py	2026-02-16 20:54:38.429213065 +0100
@@ -17,16 +17,6 @@
         'pscheduler.limitprocessor.identifier',
         'pscheduler.limitprocessor.limit',
     ],
-    install_requires=[
-        'dnspython >= 1.12.0',
-        'requests >= 2.6.0',
-        'psycopg2 >= 2.6.2',
-        'jsonschema >= 2.5.1',
-        'pyjq >= 2.2.0',
-        'python-dateutil >= 2.5.3',
-        'netifaces >= 0.5',
-        'ipaddr >= 2.1.9',
-    ],
     include_package_data=True,
     package_data={'pscheduler.limitprocessor': ['*.json']},
 
