--- a/test_subprocess32.py
+++ b/test_subprocess32.py
@@ -206,7 +206,7 @@
                      "import sys; sys.stdout.write('BDFL')\n"
                      "sys.stdout.flush()\n"
                      "while True: pass"],
-                    timeout=0.5)
+                    timeout=10.5)
         except subprocess.TimeoutExpired, exception:
             self.assertEqual(exception.output, 'BDFL')
         else:
