--- python-pyjq-d10.orig/_pyjq.pyx	2019-09-18 07:14:51.000000000 +0000
+++ python-pyjq-d10/_pyjq.pyx	2021-01-27 19:25:32.817950000 +0000
@@ -4,6 +4,7 @@
 """
 
 import os
+import sys
 
 
 class ScriptRuntimeError(Exception):
@@ -42,7 +43,7 @@
 
     jv jv_number(double)
     double jv_number_value(jv)
-    bint jv_is_integer(jv)
+    bint jv_is_integer_large(jv)
 
     jv jv_array()
     jv jv_array_sized(int)
@@ -102,8 +103,13 @@
         return True
     elif kind == JV_KIND_NUMBER:
         v = jv_number_value(jval)
-        if jv_is_integer(jval):
-            return int(v)
+        if jv_is_integer_large(jval):
+            if (sys.version_info > (3, 0)):
+                # Python 3 - All ints are arbitrarily-large
+                return int(v)
+            else:
+                # Python 2 - Has 32-bit ints and arbitrarily-large longs
+                return long(v) if v > sys.maxsize else int(v)
         return v
     elif kind == JV_KIND_STRING:
         return jv_string_value(jval).decode('utf-8')
