Description: Ensure jq doesn't statically link in libraries
Author: Harlan Lieberman-Berg <hlieberman@debian.org>
Forwarded: not-needed
--- a/Makefile.am
+++ b/Makefile.am
@@ -84,11 +84,10 @@
 
 bin_PROGRAMS = jq
 jq_SOURCES = main.c version.h
-jq_LDFLAGS = -static-libtool-libs
 jq_LDADD = libjq.la -lm
 
 if ENABLE_ALL_STATIC
-jq_LDFLAGS += -all-static
+jq_LDFLAGS = -all-static
 endif
 
 ### Tests (make check)
--- a/tests/setup
+++ b/tests/setup
@@ -9,8 +9,9 @@
 JQ=$JQBASEDIR/jq
 
 if [ -z "${NO_VALGRIND-}" ] && which valgrind > /dev/null; then
-    VALGRIND="valgrind --error-exitcode=1 --leak-check=full \
-                       --suppressions=$JQTESTDIR/onig.supp"
+    VALGRIND="libtool --mode=execute valgrind \
+    		      	       --error-exitcode=1 --leak-check=full \
+		      	       --suppressions=$JQTESTDIR/onig.supp"
     Q=-q
 else
     VALGRIND=
