diff --git a/src/jv.c b/src/jv.c
index 2427b00..9082b32 100644
--- a/src/jv.c
+++ b/src/jv.c
@@ -7,6 +7,7 @@
 #include <stdarg.h>
 #include <limits.h>
 #include <math.h>
+#include <float.h>
 
 #include "jv_alloc.h"
 #include "jv.h"
@@ -159,6 +160,18 @@ int jv_is_integer(jv j){
   return x == (int)x;
 }
 
+int jv_is_integer_large(jv j){
+  if(jv_get_kind(j) != JV_KIND_NUMBER){
+    return 0;
+  }
+  double x = jv_number_value(j);
+
+  double ipart;
+  double fpart = modf(x, &ipart);
+
+  return fabs(fpart) < DBL_EPSILON;
+}
+
 /*
  * Arrays (internal helpers)
  */
diff --git a/src/jv.h b/src/jv.h
index d111c80..39d50ca 100644
--- a/src/jv.h
+++ b/src/jv.h
@@ -63,6 +63,7 @@ jv jv_bool(int);
 jv jv_number(double);
 double jv_number_value(jv);
 int jv_is_integer(jv);
+int jv_is_integer_large(jv);
 
 jv jv_array(void);
 jv jv_array_sized(int);
