Index: pscheduler-server/api-server/pscheduler-api-server.conf.raw
===================================================================
--- pscheduler-server.orig/api-server/pscheduler-api-server.conf.raw	2018-08-24 18:25:48.000000000 +0000
+++ pscheduler-server/api-server/pscheduler-api-server.conf.raw	2018-12-18 21:45:29.390069390 +0000
@@ -16,8 +16,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Order deny,allow
-    Allow from all
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
