--- a/api-server/pscheduler-api-server.conf.raw	2019-07-23 19:41:51.565732000 +0000
+++ b/api-server/pscheduler-api-server.conf.raw	2019-07-23 19:42:41.283952000 +0000
@@ -26,7 +26,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Require all granted
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
