Index: pscheduler-server/api-server/pscheduler-api-server.conf.raw
===================================================================
--- pscheduler-server.orig/api-server/pscheduler-api-server.conf.raw	2017-11-15 10:52:31.000000000 -0700
+++ pscheduler-server/api-server/pscheduler-api-server.conf.raw	2017-11-17 14:38:52.000000000 -0700
@@ -16,8 +16,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Order deny,allow
-    Allow from all
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
