--- a/api-server/pscheduler-api-server.conf.raw	2019-08-09 00:11:58.091444233 +0000
+++ b/api-server/pscheduler-api-server.conf.raw	2019-08-09 00:12:03.605498117 +0000
@@ -26,7 +26,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Require all granted
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
