--- pscheduler-server.orig/api-server/pscheduler-api-server.conf.raw	2020-04-27 23:13:14.086176936 +0000
+++ pscheduler-server/api-server/pscheduler-api-server.conf.raw	2020-04-27 23:13:45.002506389 +0000
@@ -27,7 +27,13 @@
     WSGIProcessGroup __USER_NAME__
     WSGIApplicationGroup __USER_NAME__
 
-    Require all granted
+    <IfVersion >= 2.4>
+        Require all granted
+    </IfVersion>
+    <IfVersion < 2.4>
+        Order allow,deny
+        Allow from all
+    </IfVersion>
 
 </Directory>
 
