--- pscheduler-server.orig/utilities/backup.raw	2018-02-22 10:36:23.000000000 +0000
+++ pscheduler-server/utilities/backup.raw	2018-02-26 15:28:05.000000000 +0000
@@ -73,6 +73,7 @@
     --format plain \
     --serializable-deferrable \
     --user pscheduler \
+    --host localhost \
     pscheduler \
     > "${WORK_DIR}/database-dump"
 $INTERACTIVE && printf ' Done.\n' 1>&2 || true
--- pscheduler-server.orig/utilities/restore.raw	2018-02-26 15:27:42.000000000 +0000
+++ pscheduler-server/utilities/restore.raw	2018-02-26 15:30:11.000000000 +0000
@@ -197,7 +197,7 @@
 DB_OUTPUT="${WORK_DIR}/db-output"
 LOAD_EXIT=0
 
-postgresql-load "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
++su -l -c psql postgres < "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
     || LOAD_EXIT=$?
 
 if [ "${LOAD_EXIT}" -ne 0 ]
