--- a/utilities/backup.raw	2021-04-20 18:48:28.312528000 +0000
+++ b/utilities/backup.raw	2021-05-20 12:52:30.399639002 +0000
@@ -73,6 +73,7 @@
     --format plain \
     --serializable-deferrable \
     --user pscheduler \
+    --host localhost \
     pscheduler \
     > "${WORK_DIR}/database-dump"
 $INTERACTIVE && printf ' Done.\n' 1>&2 || true
--- a/utilities/restore.raw	2021-04-20 18:48:28.312528000 +0000
+++ b/utilities/restore.raw	2021-05-20 12:52:30.400861419 +0000
@@ -203,7 +203,7 @@
 DB_OUTPUT="${WORK_DIR}/db-output"
 LOAD_EXIT=0
 
-postgresql-load "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
+su -l -c psql postgres < "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
     || LOAD_EXIT=$?
 
 if [ "${LOAD_EXIT}" -ne 0 ]
