Index: pscheduler-server/utilities/backup.raw
===================================================================
--- pscheduler-server.orig/utilities/backup.raw	2017-11-07 16:09:11.000000000 +0000
+++ pscheduler-server/utilities/backup.raw	2017-11-27 14:44:52.000000000 +0000
@@ -73,6 +73,7 @@
     --format plain \
     --serializable-deferrable \
     --user pscheduler \
+    --host localhost \
     pscheduler \
     > "${WORK_DIR}/database-dump"
 $INTERACTIVE && printf ' Done.\n' 1>&2 || true
Index: pscheduler-server/utilities/restore.raw
===================================================================
--- pscheduler-server.orig/utilities/restore.raw	2017-11-07 16:09:11.000000000 +0000
+++ pscheduler-server/utilities/restore.raw	2017-11-27 14:44:52.000000000 +0000
@@ -193,7 +193,7 @@
 cat "${WORK_DIR}/database-dump" >> "${SQL_WORK}"
 
 DB_OUTPUT="${WORK_DIR}/db-output"
-postgresql-load "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1
+su -l -c psql postgres < "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1
 if [ $? -ne 0 ]
 then
     printf "\n\nDatabase restore failed:\n"
