--- pscheduler-server.orig/utilities/backup.raw	2019-02-13 15:40:39.361218403 +0100
+++ pscheduler-server/utilities/backup.raw	2019-02-13 15:40:39.357218340 +0100
@@ -73,6 +73,7 @@
     --format plain \
     --serializable-deferrable \
     --user pscheduler \
+    --host localhost \
     pscheduler \
     > "${WORK_DIR}/database-dump"
 $INTERACTIVE && printf ' Done.\n' 1>&2 || true
--- pscheduler-server.orig/utilities/restore.raw	2019-02-13 15:40:39.361218403 +0100
+++ pscheduler-server/utilities/restore.raw	2019-02-13 15:40:39.357218340 +0100
@@ -203,7 +203,7 @@
 DB_OUTPUT="${WORK_DIR}/db-output"
 LOAD_EXIT=0
 
-postgresql-load "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
+su -l -c psql postgres < "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
     || LOAD_EXIT=$?
 
 if [ "${LOAD_EXIT}" -ne 0 ]
