--- a/utilities/backup.raw	2021-08-26 03:07:04.625621000 +0100
+++ b/utilities/backup.raw	2021-08-26 03:17:41.011297196 +0100
@@ -73,6 +73,7 @@
     --format plain \
     --serializable-deferrable \
     --user pscheduler \
+    --host localhost \
     pscheduler \
     > "${WORK_DIR}/database-dump"
 $INTERACTIVE && printf ' Done.\n' 1>&2 || true
--- a/utilities/restore.raw	2021-08-26 03:07:04.625621000 +0100
+++ b/utilities/restore.raw	2021-08-26 03:17:41.011997226 +0100
@@ -203,7 +203,7 @@
 DB_OUTPUT="${WORK_DIR}/db-output"
 LOAD_EXIT=0
 
-postgresql-load "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
+su -l -c psql postgres < "${SQL_WORK}" > "${DB_OUTPUT}" 2>&1 \
     || LOAD_EXIT=$?
 
 if [ "${LOAD_EXIT}" -ne 0 ]
