Index: pscheduler-server/database/db-update.raw
===================================================================
--- pscheduler-server.orig/database/db-update.raw	2017-11-27 14:44:53.000000000 +0000
+++ pscheduler-server/database/db-update.raw	2017-11-27 14:51:45.000000000 +0000
@@ -18,9 +18,11 @@
 $INTERACTIVE && printf "Loading database:\n" || true
 
 $INTERACTIVE && printf '    Administrative...' || true
-postgresql-load '__DATADIR__/database-build-super.sql'
+DB_ADMIN=$(ls -v /usr/share/dbconfig-common/data/pscheduler-server/upgrade-dbadmin/pgsql/* | tail -1)
+su -l -c "psql" postgres <$DB_ADMIN >/dev/null 2>&1
 $INTERACTIVE && printf ' Done.\n' || true
 
 $INTERACTIVE && printf '    User __ROLE__...' || true
-postgresql-load --role '__ROLE__' '__DATADIR__/database-build.sql'
+DB_USER=$(ls -v /usr/share/dbconfig-common/data/pscheduler-server/upgrade/pgsql/* | tail -1)
+su -l -c "psql -U __ROLE__ pscheduler" pscheduler <$DB_USER >/dev/null 2>&1
 $INTERACTIVE && printf ' Done.\n' || true
