--- a/database/db-update.raw	2021-05-26 13:38:44.605318368 +0000
+++ b/database/db-update.raw	2021-05-26 13:38:50.249223393 +0000
@@ -20,14 +20,11 @@
 $INTERACTIVE && printf "Loading database:\n" || true
 
 $INTERACTIVE && printf '    Administrative...' || true
-postgresql-load \
-    --log-errors "${WHOAMI}-super" \
-    '__DATADIR__/database-build-super.sql'
+DB_ADMIN=$(ls -v /usr/share/dbconfig-common/data/pscheduler-server/upgrade-dbadmin/pgsql/* | tail -1)
+su -l -c "psql" postgres <$DB_ADMIN >/dev/null 2>&1
 $INTERACTIVE && printf ' Done.\n' || true
 
 $INTERACTIVE && printf '    User __ROLE__...' || true
-postgresql-load \
-    --role '__ROLE__' \
-    --log-errors "${WHOAMI}-__ROLE__"  \
-    '__DATADIR__/database-build.sql'
+DB_USER=$(ls -v /usr/share/dbconfig-common/data/pscheduler-server/upgrade/pgsql/* | tail -1)
+su -l -c "psql -U __ROLE__ pscheduler" pscheduler <$DB_USER >/dev/null 2>&1
 $INTERACTIVE && printf ' Done.\n' || true
