--- a/daemons/archiver.raw
+++ b/daemons/archiver.raw
@@ -341,7 +341,7 @@
                                        attempts = attempts + 1,
                                        last_attempt = now(),
                                        next_attempt = NULL,
-                                       diags = diags || (%s::JSONB)
+                                       diags = %s
                                      WHERE id = %s""",
                                   [ attempt, archiving_id ])
 
@@ -363,7 +363,7 @@
                                              attempts = attempts + 1,
                                              last_attempt = now(),
                                              next_attempt = NULL,
-                                             diags = diags || (%s::JSONB)
+                                             diags = %s
                                          WHERE id = %s""",
                                       [ attempt, archiving_id ])
 
@@ -389,7 +389,7 @@
                                                  attempts = attempts + 1,
                                                  last_attempt = now(),
                                                  next_attempt = %s,
-                                                 diags = diags || (%s::JSONB)
+                                                 diags = %s
                                              WHERE id = %s""",
                                           [ next, attempt, archiving_id ])
 
