Index: pscheduler-server/database/warmboot.raw
===================================================================
--- pscheduler-server.orig/database/warmboot.raw	2017-10-02 09:19:58.000000000 -0600
+++ pscheduler-server/database/warmboot.raw	2017-11-17 14:38:53.000000000 -0700
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
