Index: pscheduler-server/database/warmboot.raw
===================================================================
--- pscheduler-server.orig/database/warmboot.raw	2018-03-04 23:22:24.000000000 +0000
+++ pscheduler-server/database/warmboot.raw	2018-03-04 23:38:21.000000000 +0000
@@ -3,8 +3,7 @@
 # Warm boot the database
 #
 
-postgresql-load --role __ROLE__ <<EOF
-\c __DATABASE__
+PGPASSFILE=/etc/pscheduler/database/pgpassfile psql -U __ROLE__ __ROLE__ -h localhost <<EOF >/dev/null
 DO \$\$
 BEGIN
     PERFORM warm_boot();
